include/graph.h

```diff
@@ -127,6 +130,10 @@ class Graph {
   void set_verifier(std::unique_ptr<GraphVerifier> verifier) {
     this->verifier = std::move(verifier);
   }
+
+  // to induce a failure mid-CC
+  bool fail_round_2 = false;

```

Comment:
We should have these variables and all the other related code ifdef'd behind VERIFY_SAMPLES